code/modules/mob/living/simple_animal/abnormality/aleph/blue_star.dm

```diff
@@ -99,6 +99,8 @@
 		datum_reference.qliphoth_change(-1)
 	if(user.sanity_lost)
 		datum_reference.qliphoth_change(-1)
+	if(prob(1))
```

Comment:
All of it should be moved to variables and _abnormality.dm

Reply:
Done~